pkgs/development/embedded/openocd/default.nix

```diff
@@ -21,6 +22,14 @@ stdenv.mkDerivation rec {
   buildInputs = [ hidapi libftdi1 libusb1 ]
     ++ lib.optional stdenv.isLinux libgpiod;
 
+  patches = [
+    (fetchpatch
```

Comment:
Can you add a comment here explaining why this patch is necessary and saying when it can or should be removed?

Having it in the commit message is good, but having it right in the code can also be quite helpful.

Reply:
Sure, done.